Makefile

```diff
@@ -75,7 +75,8 @@ lint-docker:
 	@hadolint ./graders/**/Dockerfile ./workspaces/**/Dockerfile ./images/**/Dockerfile Dockerfile
 lint-actions:
 	@actionlint
-format: format-js format-python
+
+format: format-js format-python format-d2
```

Comment:
This makes `d2` a hard dependency for anyone doing local dev, even if they aren't touching the docs. Is this desired?

Reply:
I don't think it's unreasonable to have local development for linting/formatting require additional tooling. We add installation instructions for d2 as well.

Reply:
Thought about this some more: d2 seems reasonable to require being installed, actionlink/hadolint/shellcheck do not.

Reply:
We can see how this goes!